app/models/miq_server/worker_management/monitor.rb

```diff
@@ -38,14 +36,16 @@ def worker_not_responding(w)
   end
 
   def sync_workers
-    MiqWorkerType.worker_classes.each_with_object({}) do |klass, result|
-      result[klass.name] = klass.sync_workers
-      result[klass.name][:adds].each { |pid| worker_add(pid) unless pid.nil? }
+    MiqWorkerType.worker_classes.each do |klass|
+      result = klass.sync_workers
+      result[:adds].each { |pid| worker_add(pid) unless pid.nil? }
```

Comment:
Naming it result is confusing, since we aren't actually returning it anymore.  Think this is clearer to read:

```suggestion
      workers = klass.sync_workers
      workers[:adds].each { |pid| worker_add(pid) unless pid.nil? }
```

Reply:
There's an instance-var and attr_reader named `workers` so I think a local var named the same is also confusing

Reply:
Oh ok, but I still think result is confusing... `w` or `synced_workers` could also work... Naming is hard 😅

Reply:
Okay will go with synced_workers then